assets/src/components/propertiesPanel/ghostPropertiesPanel.tsx

```diff
@@ -29,21 +29,31 @@ const NoWaiverBanner = () => (
   </div>
 )
 
-const GhostPropertiesPanel = ({ selectedGhost, route }: Props) => (
-  <div className="m-ghost-properties-panel">
-    <Header vehicle={selectedGhost} route={route} />
+const GhostPropertiesPanel = ({ selectedGhost, route }: Props) => {
+  const [tabMode, setTabMode] = useState<TabMode>("status")
 
-    {hasBlockWaiver(selectedGhost) ? (
-      <BlockWaiverList blockWaivers={selectedGhost.blockWaivers} />
-    ) : (
-      <NoWaiverBanner />
-    )}
+  return (
+    <div className="m-ghost-properties-panel">
+      <Header
+        vehicle={selectedGhost}
+        route={route}
+        tabMode={tabMode}
+        setTabMode={setTabMode}
+      />
 
-    <StatusRunBlockTabs
-      vehicleOrGhost={selectedGhost}
-      statusContent={<PropertiesList vehicleOrGhost={selectedGhost} />}
-    />
-  </div>
-)
+      {hasBlockWaiver(selectedGhost) ? (
+        <BlockWaiverList blockWaivers={selectedGhost.blockWaivers} />
+      ) : (
+        <NoWaiverBanner />
+      )}
+
+      <TabPanels
```

Comment:
By moving the tabs to the header, any block waivers will now show between the tabs and the content. Is that what we want?

I realize that it might be tricky to do it the other way if we want the block waivers to scroll but the tabs to be sticky. Maybe the block waivers should live as part of the statusContent and not show in the run/block tabs?

Reply:
Right, doing it where the block waivers are above but can scroll out of sight while the tabs stick gets us back to the complex and hackish-feeling JS implementation that we were trying to avoid--or possibly back to trying those flexbox tricks from the other day.

Considering that block waivers currently do show in all three tabs, I think in the absence of a compelling reason to show them on just the status tab, that we keep them on all three tabs.

Reply:
If we want the block waivers above the tabs, and don't mind them being sticky too, we could do:
```
<div class=panel>
  <div class=sticky-wrapper>
    <Header/>
    <BlockWaivers/>
    <Tabs/>
  </div>
  <PanelContent (status or minischedule)/>
</div>
```
(moving the tabs and the sticky behavior out of the header and into the VPP/GPP)

Reply:
@skyqrose I think at this point it'd be best to ask @ingridpierre which she'd prefer:

* Keep block waivers above tabs, make them sticky.
* Block waivers go below tabs, and scroll

I'd figure the former, but perhaps not?

Reply:
@ingridpierre @skyqrose or third option, if we wanted, put the block waivers below the tabs but make them sticky.

Reply:
Another option would be to move them to the status tab only and make them not sticky.

Reply:
@skyqrose @phildarnowsky thanks for flagging this question about what to do with block waivers.

I just spoke with @ingridpierre and we agreed the following would be best:

- Put the block waivers below the tabs (specifically in the STATUS tab, the way it already is)

- Block waiver information should scroll like the rest of VPP content (making it sticky would lead to too much vertical height being consumed by sticky elements on mobile)

@skyqrose @phildarnowsky I know we already support scrolling of the RUN and BLOCK views in mini schedules, but do we need a ticket to support scrolling of the STATUS tab to support the requirements outlined above for block waivers?